### PR TITLE
Remove TagCommand

### PR DESCRIPTION
**Pull request by jovkusuma**

### CODE REVIEW

File: src/test/java/seedu/address/model/person/PersonTest.java [hunk 1]
```diff
@@ -19,11 +18,11 @@
...
-    @Test
-    public void asObservableList_modifyList_throwsUnsupportedOperationException() {
-        Person person = new PersonBuilder().build();
-        assertThrows(UnsupportedOperationException.class, () -> person.getTags().remove(0));
-    }
+    //    @Test
+    //    public void asObservableList_modifyList_throwsUnsupportedOperationException() {
+    //        Person person = new PersonBuilder().build();
+    //        assertThrows(UnsupportedOperationException.class, () -> person.getTags().remove(0));
+    //    }
```
Comment (lambraydon) [inline src/test/java/seedu/address/model/person/PersonTest.java:25]: Should we delete commented out code to adhere to the coding standard?

**Review by lambraydon — CHANGES_REQUESTED** [review]
Generally looks good with slight nitpicks for the commented out code.

File: src/test/java/seedu/address/model/person/PersonTest.java (hunk 1 above)
Comment (yihfei) [reply src/test/java/seedu/address/model/person/PersonTest.java:25]: I think we should delete as tags field are removed from Person class. If need be we could bring back the code by referencing git later on.

**Review by WinstonLeonard — APPROVED** [review]
noice

File: src/test/java/seedu/address/model/person/PersonTest.java [hunk 2]
```diff
@@ -18,13 +16,6 @@
 import seedu.address.testutil.PersonBuilder;
 
 public class PersonTest {
-
-    @Test
-    public void asObservableList_modifyList_throwsUnsupportedOperationException() {
-        Person person = new PersonBuilder().build();
-        assertThrows(UnsupportedOperationException.class, () -> person.getTags().remove(0));
-    }
-
```
Comment (yihfei) [inline src/test/java/seedu/address/model/person/PersonTest.java:27]: Resolved previous comments noice!

**Review by yihfei — APPROVED** [review]
LGTM!

File: src/test/java/seedu/address/model/person/PersonTest.java (hunk 2 above)
Comment (lambraydon) [reply src/test/java/seedu/address/model/person/PersonTest.java:27]: Great job on removing unused code!

**Review by lambraydon — APPROVED** [review]
LGTM!

**Pull request merged**